core/controlplane/config/templates/cluster.yaml

```diff
@@ -468,6 +468,12 @@ worker:
 #      kube2IamSupport:
 #        enabled: true
 #
+#      # Propagate custom CLI options to kubelet
+#      # Provided example enables docker image garbage collection as in https://kubernetes.io/docs/concepts/cluster-administration/kubelet-garbage-collection/
```

Comment:
yes please. we have many people starting playing with kube using kube-aws, providing accurate information is important. Original formulation gives impression that GC is switched off, which might cause panic :)